hack/terraform/config.tf

```diff
@@ -14,7 +14,16 @@ terraform {
     region = "ap-southeast-1"
   }
 }
-
 provider "aws" {
   region = "ap-southeast-1"
 }
+
+variable "s3_bucket_names" {
+  type = list(string)
+  default = ["download.devstream.io", "just-for-test"]
+}
+
+resource "aws_s3_bucket" "devstream_buckets" {
```

Comment:
There are a few issues with this bucket:

The objects in the bucket are public.

For example, you can download this file https://s3.ap-southeast-1.amazonaws.com/download.devstream.io/v0.4.0/argocd-darwin-amd64_0.4.0.md5 directly with the S3 URL. 

This _should not_ be allowed.

If the user uses this URL, we will pay for the data transfer fees.

Solution: [S3 static website hosting](https://docs.aws.amazon.com/AmazonS3/latest/userguide/WebsiteHosting.html) with [CORS](https://docs.aws.amazon.com/AmazonS3/latest/userguide/cors.html). We should only allow users to download the content from download.devstream.io instead of using other domain names.

Make sure Cloudflare is enabled as well.